browser-specific-failures.js

```diff
@@ -0,0 +1,221 @@
+'use strict';
+
+/**
+ * Implements a view of how many browser specific failures each engine has over
+ * time.
+ */
+
+const fetch = require ('node-fetch');
+const fs = require ('fs');
+const flags = require('flags');
+const Git = require('nodegit');
```

Comment:
This stood out to me: why is it the only capitalized import?

Reply:
It also matches https://github.com/nodegit/nodegit#api-examples.

`const Octokit = require("@octokit/rest")` is another case that I see often, there it's because you use `new Octokit(...)`.